### ISSUE TITLE
Pascal casing for demo function names

### ISSUE THREAD

**Issue by Alexey-T** [eriklins/Pascal-Bindings-For-SimpleBLE-Library#9]
adapter_on_scan_stop
adapter_on_scan_updated
etc

**Comment by Alexey-T** [comment]
and for all variables!
```
var
  adapter_identifier: PChar;
  peripheral_identifier: PChar;
  peripheral_address: PChar;
```

**Comment by eriklins** [comment]
Fixed with commit 087cbea4e23d1d6fb673ddddf305e1a343c4ca1d
